dojo/tools/hcl_appscan/parser.py

```diff
@@ -24,60 +33,85 @@ def get_findings(self, file, test):
         report = root.find("issue-group")
         if report is not None:
             for finding in report:
+                title = ""
                 description = ""
                 for item in finding:
                     match item.tag:
                         case 'severity':
-                            severity = item.text
+                            output = self.xmltreehelper(item)
+                            severity = output.strip(" ").capitalize()
                         case 'cwe':
-                            cwe = item.text
+                            cwe = int(self.xmltreehelper(item))
                         case 'remediation':
-                            remediation = item.text
+                            remediation = self.xmltreehelper(item)
                         case 'advisory':
-                            advisory = item.text
+                            advisory = self.xmltreehelper(item)
+                        case 'issue-type':
+                            title = self.xmltreehelper(item)
+                            description = description + "Issue-Type:" + title + "\n"
                         case 'issue-type-name':
-                            issuetypename = item.text
-                            description = description + "Issue-Type-Name: " + issuetypename + "\n"
+                            title = self.xmltreehelper(item)
+                            description = description + "Issue-Type-Name:" + title + "\n"
                         case 'location':
-                            location = item.text
-                            description = description + "Location: " + location + "\n"
+                            location = self.xmltreehelper(item)
+                            description = description + "Location:" + location + "\n"
                         case 'domain':
-                            domain = item.text
-                            description = description + "Domain: " + domain + "\n"
+                            domain = self.xmltreehelper(item)
+                            title += "_" + domain
+                            description = description + "Domain:" + domain + "\n"
+                        case 'threat-class':
+                            threatclass = self.xmltreehelper(item)
+                            description = description + "Threat-Class:" + threatclass + "\n"
+                        case 'entity':
+                            entity = self.xmltreehelper(item)
+                            title += "_" + entity
```

Comment:
Same comment as above

```suggestion
                            title += "_" + entity.strip()
```